automated install/basic-install.sh

```diff
@@ -84,6 +84,17 @@ if command -v apt-get &> /dev/null; then
   LIGHTTPD_CFG="lighttpd.conf.debian"
   DNSMASQ_USER="dnsmasq"
 
+  package_check_update_available() {
+    # Check if package would be upgraded.
+    # return values:
+    #  0   if no
+    #  1   if yes
+    #  >1  if more than one package containing the name
+    #      ( might still make restarting the core package necessary )
+    ${PKG_MANAGER} -u upgrade --assume-no | grep -c "$1"
+    return 0
```

Comment:
Not sure, but also this will always return 0, no matter what the grep count is. As for the grep count tripping the `set -e` flag, a `| true` at the end of line 94 should protect against that. Kind of hacky, but it works.